spec/basket_spec.rb

```diff
@@ -2,19 +2,35 @@
 
 RSpec.describe Basket do
 
-  it "creates basket" do
-    expect Basket.new.is_a? Basket
+  let(:product1) { Product.new(name: "Football", price: 79) }
+  let(:product2) { Product.new(name: "Basketball", price: 60) }
+  let(:basket1) { Basket.new([product1, product2]) }
+
+  it "calculates products sum correctly" do
```

Comment:
I think it would be more clearly if you put this `it "calculates products sum correctly"` and `it "checks if sum with vat is correct"` into a mutual context/describe or as I said before test `sum` and `sum_with_vat` seperately.